README.md

```diff
@@ -27,7 +27,7 @@ There are also a few helpers available:
 
 ## Machine setup
 
-1. Install .NET Core v2.2.6 (SDK and Runtime) [from the download archive](https://dotnet.microsoft.com/download/dotnet-core/2.2) for your system.
+1. Install .NET Core v3.0 (SDK and Runtime) [from the download archive](https://dotnet.microsoft.com/download/dotnet-core) for your system.
```

Comment:
```suggestion
1. Install .NET Core v3.0 (SDK and Runtime) [from the download archive](https://dotnet.microsoft.com/download/dotnet-core/3.0) for your system.
```

Reply:
If we're so far out of date that they've dropped 3.0 from their overall summary page, this project is probably dead anyway :)